### ISSUE TITLE
Check if emoji exists

### ISSUE THREAD

**Issue by ConcernedHobbit** [omnidan/node-emoji#16]
It would be a handy feature, especially if you're using user input

**Comment by smeijer** [comment]
What direction? Check if `:smile:` exists? Or if `😄` exists?

**Comment by omnidan** [comment]
@smeijer I think both could be interesting, maybe we could return an object, e.g. `{ emoji: true, key: false }` if it's an emoji (:smile:) and not a key (`:smile:`)

**Comment by smeijer** [comment]
I think that wouldn't be even necessary, a simple boolean return value would do it I guess. The caller is aware if he is checking a string or emoji.

**Comment by omnidan** [comment]
@smeijer maybe, I think it's a bit too much "magic" if `true` means "either some string that could mean an emoji, or an actual emoji". Especially if you want to enforce something to be a valid emoji string *only* (not an actual emoji), the separation is needed.

The object would also be more consistent with the rest of the API, e.g.:

```
emoji.random() // returns a random emoji + key, e.g. `{ emoji: '❤️', key: 'heart' }`
```

**Comment by smeijer** [comment]
So more something like 

```js
Emoji.find = (x) => ({ emoji: '❤️', key: 'heart' })
```

X can either be the emoji or the string, and the function returns undefined when no result? 

I guess we can also add a function `Emoji.isEmoji` which checks if a string or emoji is provided. 

```js
Emoji.find = (x) => ({ emoji: '❤️', key: 'heart' })
Emoji.isEmoji = (emoji) => true | false
```

The `isEmoji` can only be used by emojis known by this library. So, what is the return value of "exists", for new valid emojis, that are not (yet) defined in this library?

**Comment by omnidan** [comment]
@smeijer sounds good, maybe we can have two separate functions:

the following would return `true` for :heart: but not `:heart:`:

```js
Emoji.isEmoji = (str) => true | false
```

and this would return `true` for `:heart:` but not :heart: :

```js
Emoji.isEmojiCode = (str) => true | false
```

**Comment by smeijer** [comment]
Covered by #53.

**Comment by omnidan** [comment]
released `1.8.0` with #53 merged. you can now use `Emoji.find` and `Emoji.hasEmoji`
